src/Microsoft.Health.Dicom.Core/Models/Operations/DicomOperation.cs

```diff
@@ -29,4 +29,9 @@ public enum DicomOperation
     /// Specifies an copy operation that copies DICOM instance and metadata file in the same container.
     /// </summary>
     Copy,
+
+    /// <summary>
+    /// Specifies an delete operation that deletes DICOM instance and metadata file in the same container.
+    /// </summary>
+    Delete,
```

Comment:
nit: I suspect we'll have an async delete at some point. Perhaps this is more like `MigrationDeletion` or `MigrationEnd`?

Reply:
Done